packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -107,495 +125,283 @@ describe("calcite-input-time-picker", () => {
-    const locale = "en";
-    const numberingSystem = "latn";
-
-    const page = await newE2EPage();
-    await page.setContent(
-      `<calcite-input-time-picker step="1" lang="${locale}" numbering-system="${numberingSystem}" value="11:00:00"></calcite-input-time-picker>`,
-    );
-
-    const inputTimePicker = await page.find("calcite-input-time-picker");
-    const input = await page.find("calcite-input-time-picker >>> calcite-input-text");
-
-    expect(await input.getProperty("value")).toBe("11:00:00 AM");
-    expect(await inputTimePicker.getProperty("value")).toBe("11:00:00");
-
-    const date = new Date(0);
-    date.setHours(13);
-    date.setMinutes(59);
-    date.setSeconds(59);
-
-    const expectedValue = date.toISOString().substr(11, 8);
-    const expectedInputValue = localizeTimeString({ value: expectedValue, locale, numberingSystem });
-
-    inputTimePicker.setProperty("value", expectedValue);
-
-    await page.waitForChanges();
-
-    const inputValue = await input.getProperty("value");
-    const inputTimePickerValue = await inputTimePicker.getProperty("value");
-
-    expect(inputValue).toBe(expectedInputValue);
-    expect(inputTimePickerValue).toBe(expectedValue);
-  });
-
-  it("value displays correctly in the input when it is directly changed for a 24-hour language when a default value is present (arab lang/numberingSystem)", async () => {
-    const locale = "ar";
-    const numberingSystem = "arab";
-    const initialValue = "11:00:00";
-
-    const page = await newE2EPage();
-    await page.setContent(
-      `<calcite-input-time-picker step="1" lang="${locale}" numbering-system="${numberingSystem}" value=${initialValue}></calcite-input-time-picker>`,
-    );
-
-    const inputTimePicker = await page.find("calcite-input-time-picker");
-    const input = await page.find("calcite-input-time-picker >>> calcite-input-text");
-
-    const initialDisplayValue = localizeTimeString({ value: initialValue, locale, numberingSystem });
-    expect(await input.getProperty("value")).toBe(initialDisplayValue);
-    expect(await inputTimePicker.getProperty("value")).toBe(initialValue);
-
-    const date = new Date(0);
-    date.setHours(13);
-    date.setMinutes(59);
-    date.setSeconds(59);
-
-    const expectedValue = "13:59:59";
-    const expectedInputValue = localizeTimeString({ value: expectedValue, locale, numberingSystem });
-
-    inputTimePicker.setProperty("value", expectedValue);
-
-    await page.waitForChanges();
-
-    const inputValue = await input.getProperty("value");
-    const inputTimePickerValue = await inputTimePicker.getProperty("value");
-
-    expect(inputValue).toBe(expectedInputValue);
-    expect(inputTimePickerValue).toBe(expectedValue);
-  });
-
-  it("committing hh:mm value when step=60 with the Enter key works as expected for a 12-hour locale", async () => {
```

Comment:
Great call on that!  I will update both test loops to use milliseconds for the step so that the decimal separator is tested.

Reply:
Ok, I fixed bugs with `mk` and `bs` where neither of them were formatting with the correct decimal separator `,` and the 12 and 14 hour test loops are now testing using the full time value with milliseconds, so that's great extra coverage and addresses an issue I didn't see prior, so that was a great suggestion, thanks!